src/components/about/about-component.jsx

```diff
@@ -89,17 +91,20 @@ const AboutComponent = ({ setPageTexts, textData, VIEW , openAboutPageAnalyticsE
   return (
     <>
       <button
-        className={styles.aboutButton}
+        className={cx(className, styles.aboutButton)}
```

Comment:
Hey, shouldn't it be the other way around, to enable overriding all CSS props from `aboutButton` class?
```suggestion
        className={cx(styles.aboutButton, className)}
```